app/uk/gov/hmrc/agentservicesaccount/config/ExternalUrls.scala

```diff
@@ -61,4 +65,7 @@ class ExternalUrls @Inject() (override val configuration: Configuration) extends
   private lazy val vatThroughSoftwareExternalUrl = getConfigString("microservice.services.vat-agent-client-lookup-frontend.external-url")
   private lazy val vatThroughSoftwareStartPath = getConfigString("microservice.services.vat-agent-client-lookup-frontend.start.path")
   lazy val vatThroughSoftwareUrl: String = s"$vatThroughSoftwareExternalUrl$vatThroughSoftwareStartPath"
+
+  lazy val timeout: Int = getConfigInteger("timeoutDialog.timeout-seconds")
+  lazy val countdown: Int = getConfigInteger("timeoutDialog.timeout-countdown-seconds")
```

Comment:
not sure why these configs are in external urls but I assume it's temporary until the play upgrade